default.nix

```diff
@@ -0,0 +1,55 @@
+{ system ? builtins.currentSystem, version ? null }:
```

Comment:
Is `version` used?

Reply:
ah no, this is a relic from an old setup I was using somewhere else...